app/models/issue_importer.rb

```diff
@@ -57,7 +57,18 @@ def initialize(github_client)
   #
   # Returns a Sawyer::Resource.
   def github_issue
-    @github_issue ||= github_client.issue "#{owner}/#{repository}", number
+    @github_issue ||= begin
+      path = "/repos/#{@owner}/#{@repository}/issues/#{@number}"
+      options = {}
+
+      if defined?(@updated_at)
+        options[:headers] = {
+          "If-Modified-Since" => @updated_at.httpdate
+        }
+      end
+
+      github_client.send(:request, :get, path, nil, options)
```

Comment:
This'll end up memoizing `nil`, which means that every time `github_issue` is called it'll hit the API (assuming that the issue hasn't changed). How many times is this method called on any given instance of IssueImporter? In `#import` it's called twice in a row. Is that the only place? Would it make sense to memoize that there was a `nil` from the API, so that the app doesn't re-query in the same request?